src/render.c

```diff
@@ -201,18 +201,50 @@ int render_message_log(char *msg[7], int *color[7])
     init_pair(GREEN_PAIR, COLOR_GREEN, COLOR_BLACK);
     init_pair(YELLOW_PAIR, COLOR_YELLOW, COLOR_BLACK);
     
-    mvprintw(17, 0, "╔════════════════════════════════════════════════════════════════════════╗");
+    mvprintw(17, 0, "╔══════════════════════════════════════════════════════╗");
     for (int i = 0; i < 7; ++i) {
         mvprintw(i + 18, 0, "║ ");
         attron(COLOR_PAIR(*color[i]));
         mvprintw(i + 18, 2, "%s", msg[i]);
-        for (int j = 0; j < 71 - strlen(msg[i]); ++j) {
+        for (int j = 0; j < MAX_LOG_LEN - strlen(msg[i]); ++j) {
             printw(" ");
         } 
         attroff(COLOR_PAIR(*color[i]));
         printw("║");
     }
-    mvprintw(25, 0, "╚════════════════════════════════════════════════════════════════════════╝");
+    mvprintw(25, 0, "╚══════════════════════════════════════════════════════╝");
+    return 0;
+}
+
+int render_score_board()
+{
+    mvprintw(17, 57, "╔══════════════╗");
+    mvprintw(18, 57, "║ Score Board  ║");
+    for (int i = 19; i < 25; ++i) {
+        mvprintw(i, 57, "║              ║");
+    }
+    mvprintw(25, 57, "╚══════════════╝");
+    return 0;
+}
+
+int render_score(rcb_t *rcb) {
+    start_color();
+    init_pair(RED_PAIR, COLOR_RED, COLOR_BLACK);
+    init_pair(GREEN_PAIR, COLOR_GREEN, COLOR_BLACK);
+    gcb_t *gcb = rcb->gcb;
+    for (int i = 0; i < 2; ++i) {
+        if (gcb->score[i] > gcb->score[!i]) {
+            attron(COLOR_PAIR(GREEN_PAIR));
+        } else if (gcb->score[i] < gcb->score[!i]) {
+            attron(COLOR_PAIR(RED_PAIR));
+        }
+        mvprintw(20 + i, 59, "Player %d: %2d", i, gcb->score[i]);
+        if (gcb->score[i] > gcb->score[!i]) {
+            attroff(COLOR_PAIR(GREEN_PAIR));
+        } else if (gcb->score[i] < gcb->score[!i]) {
+            attroff(COLOR_PAIR(RED_PAIR));
+        }
+    }
```

Comment:
At first, I don't know why the score are changing colors between green and red. After reading the code I finally realize the reason.

BTW, I usually use `p` for short when indicating the current player.

Reply:
That sounds good, I'll change that.